requirements-test.txt

```diff
@@ -1,123 +1,140 @@
-    --hash=sha512:4f916143fe35d41b28efb7958ffa7e10d430d66697f8150b98aae3797a058378fd9a4c201969c155f30f88c962f78e4e75e684889d5138cd2fe60406924363e9 \
-    --hash=sha512:51652525adc37bd1af1d81933f965dba9c508838d9f759c80ca1392991515a29c2c0263264a4e175b37a6ba11dca68c354774e448b19ef1bdba96be5474d93ec \
-    --hash=sha512:ac5eedcd9425b06f44c3e9029cfc3e93a266a5b797c1b8bd3ed81cbbe6c90a163245c15f1ab5ebedf03f885edeb61ebc6b341c61e31789072f29f2a3e1b4bece \
-    ; python_version < '3.4'
-flake8==3.6.0 \
-    --hash=sha256:c01f8a3963b3571a8e6bd7a4063359aff90749e160778e03817cd9b71c9e07d2 \
-    --hash=sha256:6a35f5b8761f45c5513e3405f110a86bea57982c3b75b766ce7b65217abe1670
+    --hash=sha256:2d81cbbe0e73112bdfe6ef8576f2238f2ba27dd0d55752a776c41d38b7da2850 \
+    --hash=sha256:644837f692e5f550741432dd3f223bbb9852018674981b1664e5dc339387588a \
+    --hash=sha256:6bd0f6ad48ec2aa117d3d141940d484deccda84d4fcd884f5c3d93c23ecd8c79 \
+    --hash=sha256:8ad8c4783bf61ded74527bffb48ed9b54166685e4230386a9ed9b1279e2df5b1
 flake8-docstrings==1.3.0 \
-    --hash=sha512:b2807bd0b68071f5ea8f1de545826692d8c97c14f5f27132a2c5348a4f0599773345305208d80e6302d6ae7c714e1575c30012d50e0996ec86549df7c3278a1c \
-    --hash=sha512:c90bdcafc84f2940c401c8b22a9cfb96007e6c0b63465e611f5031735a85f1fa7f65f5d9ca898bb98e69966758e9e3ef5b907577ea0b1e925285572ed6f2f954
+    --hash=sha256:4e0ce1476b64e6291520e5570cf12b05016dd4e8ae454b8a8a9a48bc5f84e1cd \
+    --hash=sha256:8436396b5ecad51a122a2c99ba26e5b4e623bf6e913b0fea0cb6c2c4050f91eb
 flake8-polyfill==1.0.2 \
-    --hash=sha512:5d5d9c487e3fcfa3a51d226ca81b8c559e2dffaa657f902d4afc47551677c0e889bf168297a56e816eead2aa45e8a9f4ca1674e8f7c9dc653bb2a01a95132e28 \
-    --hash=sha512:767a599aacbe87328b3e36bd85e0841870760ed9bd95dd3a4a9084edc6f0ae89f8203d565c2b075f16f1db21b647c17b2aa59b08e4702109d7e5f79f36d9f3fa
+    --hash=sha256:12be6a34ee3ab795b19ca73505e7b55826d5f6ad7230d31b18e106400169b9e9 \
+    --hash=sha256:e44b087597f6da52ec6393a709e7108b2905317d0c0b744cdca6208e670d8eda \
+    # via flake8-docstrings
+flake8==3.7.5 \
+    --hash=sha256:c3ba1e130c813191db95c431a18cb4d20a468e98af7a77e2181b68574481ad36 \
+    --hash=sha256:fd9ddf503110bf3d8b1d270e8c673aab29ccb3dd6abf29bae1f54e5116ab4a91
 funcsigs==1.0.2 \
-    --hash=sha512:4e583bb7081bb1d6c0fe5a6935ca03032d562b93ef5c11b51a72ce9e7ac12902451cc2233c7e6f70440629d88d8e6e9625965ee408100b80b0024f3a6204afda \
-    --hash=sha512:8f3d521413af262085cecbe12a41cf1ad36d42efe4c7c892aca58240e1412c9380da72ec159a499a8eaa18bf5013a5892cff58b240b49f3cf4d9428caca9cfe2 \
-    ; python_version < "3.0"
+    --hash=sha256:330cc27ccbf7f1e992e69fef78261dc7c6569012cf397db8d3de0234e6c937ca \
+    --hash=sha256:a7bb0f2cf3a3fd1ab2732cb49eba4252c2af4240442415b4abce3b87022a8f50 ; python_version < "3.0"
+# For an unknown reason, putting the python_version statement at the end doesn't work
+functools32==3.2.3.post2 ; python_version < "3.0" \
```

Comment:
Does this error seem familiar to you, @Callek ? I've never seen this before. The pip bug tracker doesn't show anything

Reply:
I've never seen this either, so not sure... odd.